rct229/ruleset_functions/baseline_systems/is_baseline_system_1.py

```diff
@@ -0,0 +1,101 @@
+    is_baseline_system_1_a,
+)
+from rct229.ruleset_functions.baseline_systems.is_baseline_system_1_c import (
+    is_baseline_system_1_c,
+)
+
+HEATING_SYSTEM = schema_enums["HeatingSystemOptions"]
+
+
+def is_baseline_system_1(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
+    """
+    Get either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates whether the HVAC system is
+    ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW), system 1b (system 1 with
+    purchased heating), system 1c (system 1 with purchased CHW and purchased HW).
+
+    Parameters
+    ----------
+    rmi_b JSON, To evaluate if the hvac system is modeled as either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 in the B_RMD.
+    hvac_b_id String, The id of the hvac system to evaluate.
+    terminal_unit_id_list List<String>, list of terminal unit IDs associated with the HVAC system to be evaluated. These are
+                            sent to this function from the master get_baseline_system_types function.
+    zone_id_list  List<String>, list of zone IDs associated with the HVAC system to be evaluated. These are sent to this
+                        function from the master get_baseline_system_types function.
+
+    Returns The function returns either Sys-1, Sys-1a, Sys-1b, Sys-1c, or Not_Sys_1 string output which indicates
+    whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1 (PTAC), system 1a (system 1 with purchased CHW),
+    system 1b (system 1 with purchased heating), system 1c (system 1 with purchased CHW and purchased HW). -------
+
+    """
+
+    is_baseline_system_1_str = HVAC_SYS.UNMATCHED
+    # Get the hvac system
+
+    if is_baseline_system_1_c(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
+        is_baseline_system_1_str = HVAC_SYS.SYS_1C
+    elif is_baseline_system_1_a(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
+        is_baseline_system_1_str = HVAC_SYS.SYS_1A
+    else:
+        hvac_b = find_exactly_one_hvac_system(rmi_b, hvac_b_id)
+
+        # check if the hvac system has the required sub systems for system type 1
+        has_required_sys = (
+            hvac_b.get("preheat_system") is None
+            or hvac_b["preheat_system"].get("heating_system_type") is None
+            or hvac_b["preheat_system"]["heating_system_type"] == HEATING_SYSTEM.NONE
+        )
+
+        are_sys_data_matched = (
+            has_required_sys
+            # sub functions handles missing required sys, and return False.
+            and is_hvac_sys_heating_type_fluid_loop(rmi_b, hvac_b_id)
+            and is_hvac_sys_fan_sys_cv(rmi_b, hvac_b_id)
+            and does_hvac_system_serve_single_zone(rmi_b, zone_id_list)
+            and does_each_zone_have_only_one_terminal(rmi_b, zone_id_list)
+            and are_all_terminal_heat_sources_none_or_null(rmi_b, terminal_unit_id_list)
```

Comment:
`are_all_terminal_cool_sources_none_or_null` is in the RDS but missing from here.

Reply:
Good catch, I added this to the list.